include/nbl/asset/ICPUPipelineLayout.h

```diff
@@ -52,37 +49,34 @@ class ICPUPipelineLayout : public IAsset, public IPipelineLayout<ICPUDescriptorS
 
         core::smart_refctd_ptr<IAsset> clone(uint32_t _depth = ~0u) const override
         {
-            std::array<core::smart_refctd_ptr<ICPUDescriptorSetLayout>,DESCRIPTOR_SET_COUNT> dsLayouts;
+            std::array<core::smart_refctd_ptr<ICPUDescriptorSetLayout>, DESCRIPTOR_SET_COUNT> dsLayouts;
             for (size_t i = 0ull; i < dsLayouts.size(); ++i)
-                dsLayouts[i] = (m_descSetLayouts[i] && _depth > 0u) ? core::smart_refctd_ptr_static_cast<ICPUDescriptorSetLayout>(m_descSetLayouts[i]->clone(_depth-1u)) : m_descSetLayouts[i];
+                dsLayouts[i] = (m_descSetLayouts[i] && _depth > 0u) ? core::smart_refctd_ptr_static_cast<ICPUDescriptorSetLayout>(m_descSetLayouts[i]->clone(_depth - 1u)) : m_descSetLayouts[i];
 
             auto cp = core::make_smart_refctd_ptr<ICPUPipelineLayout>(
-                std::span<const asset::SPushConstantRange>{m_pushConstantRanges->begin(),m_pushConstantRanges->end()},
-                std::move(dsLayouts[0]),std::move(dsLayouts[1]),std::move(dsLayouts[2]),std::move(dsLayouts[3])
```

Comment:
I have no idea why but this suddenly complains about `cannot access protected member declared in class 'nbl::asset::IPipelineLayout<nbl::asset::ICPUDescriptorSetLayout>` ?